recipe/meta.yaml

```diff
@@ -38,9 +38,14 @@ source:
       # work-around for https://github.com/grpc/grpc/issues/36654
       - patches/0010-set-Wl-undefined-dynamic_lookup-for-upb_-json-textfo.patch
       - patches/0011-add-missing-GRPC_DLL-macro.patch
-  - url: https://pypi.io/packages/source/g/grpcio-tools/grpcio_tools-{{ version }}.tar.gz
-    sha256: d9657f5ddc62b52f58904e6054b7d8a8909ed08a1e28b734be3a707087bcf004
+      - patches/0012-Build-protobufs-for-tests-only-if-gRPC_BUILD_TESTS.patch
+      - patches/0013-Fixed-NAN-INFINITY-definitions-to-work-on-Windows-11.patch
+  - url: https://pypi.org/packages/source/g/grpcio-tools/grpcio_tools-{{ version }}.tar.gz
+    sha256: 737804ec2225dd4cc27e633b4ca0e963b0795161bf678285fab6586e917fd867
     folder: grpcio_tools
+    patches:
+      # We can reuse this patch as we're patching the same third_party folder
+      - grpcio-patches/0001-Fixed-NAN-INFINITY-definitions-to-work-on-Windows-11.patch
```

Comment:
If we say we can reuse the patch, shouldn't we actually do it? No strong feelings either way, just curious that the patch was fully duplicated despite that comment.